testing/freerdp2/APKBUILD

```diff
@@ -0,0 +1,110 @@
+# Maintainer: Valery Kartel <valery.kartel@gmail.com>
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+pkgname=freerdp2
+_pkgname=FreeRDP
+pkgver=2.0.0_rc0
+_pkgver=${pkgver/_rc/-rc}
+pkgrel=0
+pkgdesc="A Remote Desktop Protocol Implementation"
+url="http://www.freerdp.com/"
+arch="all"
+options="!check"
+license="ASL-2.0"
+depends=""
+makedepends="cmake linux-headers alsa-lib-dev cups-dev eudev-dev gsm-dev
```

Comment:
You should move the `-dev` stuff here to `depends_dev` where needed (e.g. if packages compiling against its headers also need the headers of those libraries), and include `$depends_dev` in `makedepends`.